index.html

```diff
@@ -1,19 +1,89 @@
 <!doctype html>
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
+  <html lang="en">
 
-  <title>My Resume</title>
+<!--Head Start-->
+    <head>
 
-  <link rel="stylesheet" href="css/index.css">
+      <title>My Resume</title>
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+      <meta charset="utf-8">
+    
+      <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+      <link rel="stylesheet" href="css/index.css">
+    
+      <meta-name="viewport" content="width=device-width, initial-scale=1">
 
-<body>
-  <h1>My Resume</h1>
-</body>
-</html>
\ No newline at end of file
+      <!--[if lt IE 9]>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+      <![endif]-->
+    </head>
+<!--Head End-->
+    
+<!--Header Start-->
```

Comment:
good job with using the comments and white space to make your html easy to read and review. Keep this habit going.